src/Sylius/Bundle/AdminBundle/Resources/views/Product/Attribute/attributeValues.html.twig

```diff
@@ -0,0 +1,50 @@
+{% set subject = metadata.name|replace({'_attribute': ''}) %}
+{% for attributeId, form in forms %}
+<div class="attribute" data-id="{{ attributeId }}">
+    {% set id = form.vars.label|replace(' ', '_')|lower %}
+    {% if 'type_checkbox' in form.vars.cache_key %}
+        {{ _self.checkboxField(form, count, id, subject, metadata) }}
+    {% else %}
+        {{ _self.textField(form, count, id, subject, metadata) }}
+    {% endif %}
+    <input type="hidden"
+           name="{{ metadata.applicationName }}_{{ subject }}[attributes][{{ count }}][attribute]"
+           id="{{ metadata.applicationName }}_{{ subject }}_attributes_{{ count }}_attribute"
+           value="{{ attributeId }}"/>
+    {% set count = count+1 %}
+</div>
+{% endfor %}
+
+{% macro formField(item, count, id, prefix, subject, applicationName) %}
```

Comment:
I'm not sure... This macro is used only here, maybe we could separate it if it'll be used somewhere else?